src/experiment.js

```diff
@@ -446,9 +446,9 @@ allImagesShown.forEach(function(image) {
 const completionCode = {
   type: "instructions",
   pages: [
-    "<div><p>You MTurk completetion code is " +
+    "<div><p>Your MTurk completetion code is " +
       param.completionCode +
-      " </p></div>"
+      " </p><p>Press J to download experiment data.</p></div>"
```

Comment:
I don't think that this should be displayed- the participant shouldn't know what key to press to download the data. Also, we should probably pick a different, more random key since the participant has been pressing J the entire experiment.

Reply:
Great catch - will fix tonight